portfolio/src/main/webapp/style.css

```diff
@@ -11,15 +11,43 @@
 h1 {
 	color: white;
 	text-align: center;
+	font-family:"Courier New", Courier, monospace;
+	font-size: 60px;
 }
 
 p {
 	color: white;
 	text-align: center;
+	font-family: "Courier New", Courier, monospace;
+	font-size: 30px;
 }
 
 img {
 	display: block;
   	margin-left: auto;
 	margin-right: auto;
 }
+
+ul { 
+	list-style-type: none;
+	margin: 0;
+	padding: 0;
+	overflow: hidden;
+	background-color: #51667b;
```

Comment:
Style Nit: Alphabetize declarations.

From style guide:
Put declarations in alphabetical order in order to achieve consistent code in a way that is easy to remember and maintain.

Ignore vendor-specific prefixes for sorting purposes. However, multiple vendor-specific prefixes for a certain CSS property should be kept sorted (e.g. -moz prefix comes before -webkit).

ex:
background: fuchsia;
border: 1px solid;
-moz-border-radius: 4px;
-webkit-border-radius: 4px;
border-radius: 4px;
color: black;
text-align: center;
text-indent: 2em;